core/templates/pages/exploration-editor-page/translation-tab/translation-tab.directive.ts

```diff
@@ -81,19 +81,20 @@ angular.module('oppia').directive('translationTab', [
             TranslationTabActiveModeService,
             UserExplorationPermissionsService) {
           var ctrl = this;
-          ctrl.directiveSubscriptions = new Subscription();
           var _ID_TUTORIAL_TRANSLATION_LANGUAGE =
-            '#tutorialTranslationLanguage';
+          '#tutorialTranslationLanguage';
```

Comment:
Please end the beginning of multi-line expressions with a parenthesis, like so:

```suggestion
          var _ID_TUTORIAL_TRANSLATION_LANGUAGE = (
            '#tutorialTranslationLanguage');
```

Here and elsewhere.

Reply:
Done